deploy/examples/upstream.operatorsource.cr.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: marketplace.redhat.com/v1alpha1
```

Comment:
Should we keep both example `OperatorSource` CRs? Having both might suggest that both should be applied, which I don't think we want.

Reply:
@SamiSousa https://github.com/operator-framework/operator-marketplace/pull/112#discussion_r257819134

I think we decided to keep both examples and direct users to the appropriate example in the appropriate section/discussion.